common/src/main/java/io/pravega/common/concurrent/ExecutorServiceHelpers.java

```diff
@@ -66,6 +66,17 @@ public void uncaughtException(Thread t, Throwable e) {
      * @return a thread factory
      */
     public static ThreadFactory getThreadFactory(String groupName) {
+        return getThreadFactory(groupName, null);
```

Comment:
Default priority can be `java.lang.Thread#NORM_PRIORITY`

Reply:
Done.